root/artist/ArtistSubscribers.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l, ln} from '../static/scripts/common/i18n';
+import EditorLink from '../static/scripts/common/components/EditorLink';
+
+import ArtistLayout from './ArtistLayout';
+
+type Props = {|
+  +canonicalURL: string,
+  +entity: ArtistT,
+  +privateEditors: number,
+  +publicEditors: $ReadOnlyArray<EditorT>,
+  +subscribed: boolean,
+|};
+
+const ArtistSubscribers = ({
+  canonicalURL,
+  entity,
+  privateEditors,
+  publicEditors,
+  subscribed,
+}: Props) => (
+  <ArtistLayout artist={entity} canonicalURL={canonicalURL} page="subscribers" title={l('Subscribers')}>
+    <h2>{l('Subscribers')}</h2>
+
+    {(publicEditors.length || (privateEditors > 0)) ? (
+      <>
+        <p>{ln(
+          'There is currently {num} user subscribed to {artist}:',
+          'There are currently {num} users subscribed to {artist}:',
+          publicEditors.length + privateEditors,
+          {
+            artist: entity.name,
+            num: publicEditors.length + privateEditors,
+          },
+        )}
+        </p>
+
+        <ul>
+          {publicEditors.map(editor => (
+            <li key={editor.id}>
+              <EditorLink editor={editor} />
+            </li>
+          ))}
+          {publicEditors.length && (privateEditors > 0) ? (
+            <li>{ln(
```

Comment:
Nitpick, `<li>` should be on its own line unless the entire thing fits on one line. I committed 916c4362d798048f42ee1d69cbdd37f7e8d2a392 so that you can just run `eslint --fix` to have it correct automatically.